### DISCUSSION TITLE
PDS tornado watches

### DISCUSSION THREAD

**Discussion by JJKraw** [weather-gov/api#597] [Q&A]
There is currently a tornado watch (575) issued by the Storm Prediction Center as a "Particularly Dangerous Situation"

https://www.spc.noaa.gov/products/watch/ww0575.html

Fetching the alert from the API, there is no mention of this important attribute.

https://api.weather.gov/alerts/urn:oid:2.49.0.1.840.0.3435bcfecffd106fab3231067db4e804cea3a251.001.2

Is there something missing from the API to convey a PDS watch? Is it an issue with the text from the issuing office (Jackson, MS)? Something else?

Thanks in advance.

**Comment by scadergit** [comment]
While those two products might be tracking the same event, they are not the same product (the SPC PDS does not contain a VTEC). I searched our entire catalog of CAP XML and did not find any CAP product with that phrase, so it makes me wonder if upstream creates CAP for SPC products. I will ask around tomorrow to see if I can confirm.

**Reply by scadergit** [reply]
I confirmed with the upstream provider that the NWS only creates CAP for weather offices and not centers (e.g. SPC).

**Reply by JJKraw** [reply]
So it's a matter of getting the offices to include this text in their watch alerts? Seems that if the PDS tag is being done for the public benefit that it ought to make it into the CAP alerts. (Looking at this from an end-user / public perspective - how it gets there doesn't matter to most people, just that it's there).

**Reply by scadergit** [reply]
I would agree--a good discussion for your local office (it's higher-up than us). In general, the NWS receives text products from offices/centers and converts them to CAP. The NWS filters quite a bit out during that effort for various reasons. Their long-term strategy is to allow offices/centers to create CAP directly, which would all be fed into the dissemination platform (aka, the API).
